core/src/Pos/Core/Util/LogSafe.hs

```diff
@@ -71,268 +55,17 @@ module Pos.Core.Util.LogSafe
 -- what's going on.
 import           Universum
 
-import           Control.Monad.Trans (MonadTrans)
-import           Data.Foldable (Foldable, length, null)
-import           Data.List (isSuffixOf)
-import           Data.Reflection (Reifies (..), reify)
-import           Data.Text.Lazy.Builder (Builder)
-import           Formatting (bprint, build, fconst, later, mapf, (%))
 import qualified Formatting.Buildable
-import           Formatting.Internal (Format (..))
-import qualified Language.Haskell.TH as TH
-import           System.Wlog (CanLog (..), HasLoggerName (..), Severity (..),
-                     logMCond)
-import           System.Wlog.LogHandler (LogHandlerTag (HandlerFilelike))
 
-import           Pos.Core (Timestamp)
 import           Pos.Core.Common (Address, Coin)
```

Comment:
the reason for why we keep this file: provide Buildable instances for types defined outside 'util'